hardhat.config.ts

```diff
@@ -30,7 +30,11 @@ const config: HardhatUserConfig = {
   },
   networks: {
     hardhat: {
-      deploy: ["deploy/core", "deploy/test"],
+      deploy: 'deploy',
```

Comment:
No arguments regarding the forking from mainnet strategy to test, but we still may want to consider a testnet deployment in the coming weeks and how we would interact with that.   Will leave that up to you, but we may need a mock fox token and a mock reactor in that case.